src/locales/cs.ts

```diff
@@ -0,0 +1,798 @@
+  //   willibrord_of_utrecht_bishop: 'Saint Willibrord, Bishop',
+  //   winefride_of_flintshire_virgin: 'Saint Winefride, Virgin',
+    wolfgang_of_regensburg_bishop: 'Sv. Wolfganga, biskupa',
+  //   wulstan_of_worcester_bishop: 'Saint Wulstan, Bishop',
+    zdislava_of_lemberk: 'Sv. Zdislavy',
+  //   zygmunt_gorazdowski_priest: 'Saint Zygmunt Gorazdowski, Priest',
+  //   sigmund_felix_felinski_bishop: 'Saint Sigmund Felix Feliński, Bishop',
+  //   holy_child_of_cebu: 'Holy Child of Cebú',
+    seven_holy_founders_of_the_servite_order: 'Sv. Alexia a druhů, řeholníků',
+  //   seven_martyred_nuns_from_the_franciscan_missionaries_of_mary: 'Seven Martyred Nuns from the Franciscan Missionaries of Mary',
+  //   shipwreck_of_saint_paul_apostle: 'Shipwreck of Saint Paul, Apostle',
+  //   dunstan_of_canterbury_bishop: 'Saint Dunstan, Bishop',
+  //   sunday_of_the_word_of_god: 'Sunday of the Word of God',
+    passion_of_saint_john_the_baptist: 'Umučení sv. Jana Křtitele',
+  //   english_martyrs: 'English Martyrs',
+  //   five_wounds_of_the_lord: 'Five Wounds of the Lord',
+    most_holy_name_of_jesus: 'Nejsvětějšího Jména Ježíš',
+  //   six_welsh_martyrs_and_companions: 'Six Welsh Martyrs and Companions',
+  //   translation_of_the_relics_of_saint_stephen_of_hungary: 'Translation of the Relics of Saint Stephen of Hungary',
+  //   vietnamese_martyrs: 'Vietnamese Martyrs',
+    visitation_of_mary: 'Navštívení Panny Marie',
+  //   waitangi_day: 'Waitangi Day',
+  //   francis_xavier_seelos_priest: 'Blessed Francis Xavier Seelos, priest',
+    immaculate_conception_of_mary_patroness_of_the_usa: 'Panny Marie, počaté bez poskvrny prvotního hříchu, patronky USA',
+  //   peter_chanel_priest_patron_of_oceania: 'Saint Peter Chanel, Proto-martyr of Oceania and Patron of Oceania',
+  //   john_brenner_priest: 'Blessed John Brenner, Priest and Martyr',
+  //   paul_of_thebes_hermit: 'Blessed Paul of Thebes, Hermit',
+  //   eusebius_of_esztergom_priest: 'Blessed Eusebius of Esztergom, Priest',
+  //   pavel_peter_gojdic_bishop: 'Blessed Pavel Peter Gojdič, Bishop and Martyr',
+  //   szilard_bogdanffy_bishop: 'Blessed Szilárd Bogdánffy, Bishop and Martyr',
+  //   ladislaus_batthyany_strattmann: 'Blessed Ladislaus Batthyány-Strattmann',
+  //   zoltan_lajos_meszlenyi_bishop: 'Blessed Zoltán Lajos Meszlényi, Bishop and Martyr',
+  //   ceferino_gimenez_malla_martyr: 'Blessed Ceferino Giménez Malla, Martyr',
+  //   john_scheffler_bishop: 'Blessed John Scheffler, Bishop and Martyr',
+    our_lady_of_sorrows_patroness_of_slovakia: 'Panny Marie Bolestné, patronky Slovenska',
+  //   george_of_lydda_martyr_patron_of_england: 'Saint George, Martyr and Patron of England',
+    benedict_of_nursia_abbot_patron_of_europe: 'Sv. Benedikta, opata, patrona Evropy',
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe: 'Sv. Cyrila, mnicha, a Metoděje, biskupa, spolupatronů Evropy',
```

Comment:
patrons of Europe are in Czech always all called "patrons", never "copatrons" (spolupatroni).

Moreover, you don't use the title everywhere the same way, evoking a (false) impression that there is a hierarchy among "patrons of Europe" ("patrons" superior to "copatrons").

Reply:
> patrons of Europe are in Czech always all called "patrons", never "copatrons" (spolupatroni).

It might be so, however, they are _co-patrons_ ([src](http://www.vatican.va/content/john-paul-ii/en/motu_proprio/documents/hf_jp-ii_motu-proprio_01101999_co-patronesses-europe.html)).

> Moreover, you don't use the title everywhere the same way

What do you mean by this? If I made a mistake somewhere, just tell me, I’ll fix it. Only St Benedict of Nursia is the (principal) patron of Europe, Sts Cyril and Methodius, St Bridget of Sweden, St Catherine of Siena and St Edith Stein are co-patron(esse)s.

Reply:
> Only St Benedict of Nursia is the (principal) patron of Europe, Sts Cyril and Methodius, St Bridget of Sweden, St Catherine of Siena and St Edith Stein are co-patron(esse)s.

That's surprising for me. Eventually I'll try to read a bit more on this topic and either correct my misconception or refute yours. For the time being please consider my comment starting this thread void.

Reply:
I am not a theologian nor liturgist, just a mere enthusiast, who uses the Internet as much as possible to get the information I need. :smiley: Therefore, don’t take the information I provide you as 100 % correct. :smiley:

> Eventually I'll try to read a bit more on this topic and either correct my misconception or refute yours.

Please do so. I might learn something more from you, however, I think that those patronage divisions (or hierarchy) are correct. There might be even more than one (principal) patronages for a particular territory (e.g. a country).